flocker/acceptance/endtoend/test_dataset.py

```diff
@@ -10,6 +10,7 @@
 from twisted.trial.unittest import TestCase
 
 from ...common import loop_until
+from ...testtools import flaky
 
 from ..testtools import (
     require_cluster, require_moving_backend, create_dataset, DatasetBackend
```

Comment:
this is confusing,
there are now two 'testtools' modules

is the intent to depreceate flocker/flocker/acceptance/testtools.py in future JIRAs ?